src/ansiblelint/rules/args.py

```diff
@@ -105,6 +105,15 @@ def matchtask(
             return []
 
         loaded_module = load_module(module_name)
+        if loaded_module.plugin_resolved_path.endswith(".ps1"):
+            _logger.warning(
```

Comment:
No need to log anything about it, it will spam output and confuse users. You can keep a code comment stating that `ps1` modules are unlikely to be executable on our platform.

Still I am quite curious if we could install pwsh on linux to be able to execute these, it would be an interesting exercise.